### ISSUE TITLE
Bug Report: Subtitle Cleaning Script Timing Issue

### ISSUE THREAD

**Issue by EmilianoNeme** [KBlixt/subcleaner#60]
### Summary:
The subtitle cleaning script incorrectly adjusts the end time of subtitle lines when two consecutive lines have identical text but different timings. This results in the first subtitle line’s end time being changed to match the end time of the subsequent line.

### Steps to Reproduce:
Run the subtitle cleaning script with a subtitle file in SRT format that contains two lines with identical text but different timings.
Example:

### Original Subtitle File:
```
1
00:00:01,000 --> 00:00:03,000
Hello!!!

2
00:00:08,000 --> 00:00:10,000
Hello!!!
```

### After Running the Script:
```
1
00:00:01,000 --> 00:00:10,000
Hello!!!

2
00:00:08,000 --> 00:00:10,000
Hello!!!
```

### Expected Behavior:
The subtitle lines should retain their original timing:
Line 1: 00:00:01,000 --> 00:00:03,000
Line 2: 00:00:08,000 --> 00:00:10,000

### Actual Behavior:
The script incorrectly changes the end time of the first subtitle line to match the end time of the second subtitle line.

**Comment by KBlixt** [comment]
Hi!

thank you for letting me know. Sorry for the late reply. summer times. 

This is now fixed. feel free to verify but I'm very confident that this is fixed. :)

The cause was a simple reversed delta check.

the part that malfunction is supposed to check for identical blocks that are within one frame of each other and merge them if that is the case. I reversed the math in the calculation of time gap between the block and thus it became a negative number and thus always smaller than 1/30th of a second. and thus went a bit haywire.
